org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/service/ServiceExtension.java

```diff
@@ -62,57 +62,71 @@ public ServiceExtension() {
 	}
 
 	@Override
-	protected boolean supportsType(TargetType targetType, ExtensionContext extensionContext)
+	protected boolean supportsType(TargetType targetType, ExtensionContext extensionContext) {
+		return true;
+	}
+
+	private Type extractServiceType(TargetType targetType, InjectService injectService)
 		throws ParameterResolutionException {
-		Type serviceType;
+		Type valueType, upperBoundType = null;
```

Comment:
I don't care for this style of variable declaration. It is non-obvious since you declare multiple variables and initialize one(?). Better to have one variable declaration per line.